gnovm/tests/files/var18.gno

```diff
@@ -0,0 +1,8 @@
+package main
+
+func main() {
+	var a, b = 1
+}
+
+// Error:
+// main/files/var18.gno:3: assignment mismatch: 2 variable(s) but 1 value(s)
```

Comment:
The line isn't good. It will be fixed at: https://github.com/gnolang/gno/pull/2221